File: C++17/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.cc
#include <iostream>
#include <string>

using namespace std;

int main (void) {
    int n, count = 1;

    while (true) {
        cin >> n;
        if (n == 0) break;

        for (int i = 0; i < n; i++) {
            int temp;
            cin >> temp;
        }

        cout << "Case " << count << ": Sorting... done!" << endl;
        count++;
    }

    return 0;
}

File: Java/백준/Bronze/23235. The Fastest Sorting Algorithm In The World/The Fastest Sorting Algorithm In The World.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        int count = 1;

        while (true) {
            int n = sc.nextInt();
            if (n == 0) break;

            for (int i = 0; i < n; i++) {
                sc.nextInt();
            }

            System.out.printf("Case %d: Sorting... done!\n", count);
            count++;
        }

        sc.close();
    }
}